symbolic-symcache/src/cache.rs

```diff
@@ -443,17 +443,17 @@ impl<'a> LineInfo<'a> {
     }
 
     /// The compilation directory of the function.
-    pub fn compilation_dir(&self) -> &'a str {
+    pub fn compilation_dir<'b>(&'b self) -> &'a str {
         self.comp_dir
     }
 
     /// The base dir of the current line.
-    pub fn base_dir(&self) -> &str {
+    pub fn base_dir<'b>(&'b self) -> &'a str {
```

Comment:
Thanks, this was oversight. The named lifetime `'b` is redundant in this case, however.